Foundation/Bundle.swift

```diff
@@ -53,8 +53,16 @@ open class Bundle: NSObject {
         self.init(path: url.path)
     }
     
-    public init(for aClass: AnyClass) { NSUnimplemented() }
-    
+#if os(Android)
+    public convenience init(for aClass: AnyClass) {
+        self.init(path: Bundle.main.bundlePath)!
```

Comment:
The goal is compatibility. If at all possible OSS code should run unmodified.

Reply:
I think we need to get a bit of perspective here. The code currently executes NSUnimplemented() isn’t much use. This is a pragmatic Android-specific step up in functionality that returns “a bundle” if this method happens to be called rather than crashing the app.

Reply:
Alamofire is relying on an assumption that isn't true on all platforms, and its OSS nature makes it trivial to propose a patch upstream. That's up to Alamofire to fix, not to Foundation to work around, and I think it's safe to say the core team is in broad agreement here.